src/main/resources/webapp/webapp.js

```diff
@@ -55,7 +55,10 @@ router.post("/api/graphql", function (req) {
 router.get("/manifest.json", function(req) {
     return manifest.get(req);
 });
-
+router.get("/leagues/{name}", defaultRoute);
+router.get("/players/{name}", defaultRoute);
+router.get("/teams/{name}", defaultRoute);
+router.get("/games/{id}", defaultRoute);
 router.get("/", defaultRoute);
```

Comment:
Just leaving $0.02 here in case it's ever an issue - [here](https://developer.enonic.com/docs/static-asset-lib/master#example-webapp) is an example of using lib-router v3.+ to handle webapp root requests differently when the URL doesn't end with a trailing slash.